### PR TITLE
feat: Added Git Credentials in Footer and Open issue in Header

### LINKED ISSUE #22: feat: Github Issues tracker + Feedback button
lead: @XavierChanth 
**Is your feature request related to a problem? Please describe.**

- [ ] A feedback button accessible on all pages to quickly open a github ticket against atsign.dev. 
- [ ] Add Git/github information and links to the footer of the site.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by PriyanshuRj**
**- What I did**
fixes #22 
I created a Link to open an issue on the atsign documentation repository and added details for the last commit in the footer as specified.
`Last updated on <date>`
`By <author> in <short commit id>`  (commit id can link to the commit in GitHub)
**- How I did it**

**- How to verify it**
After running the Documentation site on localhost, you can check the header and the footer of the site to verify the same.
image
image

**- Description for the changelog**
Updated the header of the site to add a open issue features and added details of last commit in the footer.

### CODE REVIEW

File: layouts/partials/main/footer.html [hunk 1]
```diff
@@ -2,12 +2,12 @@
     <div class="footer-logo">
         {{ $image := resources.Get "img/branding/dark/atsign-dark.png" }}
         {{ with $image }}
-            <img class="toggle-light" src="{{ .RelPermalink }}" width="124em">
```
Comment (xavierchanth) [inline layouts/partials/main/footer.html]: Please avoid introducing unnecessary formatting changes.

**Review by xavierchanth — CHANGES_REQUESTED** [review]
Hi @PriyanshuRj, thanks for your solution. There are a few more improvements I would like to see included in your solution:

1. This solution is only able to get the latest commit on the repo, It would be nice to see this information based on the page that you are on. Most times, the page that you are viewing will not have been modified in the last commit.
2. I would rather this information be part of the hugo build process, this would allow this information to be served statically instead of through a client side script (much faster loads and no unnecessary api calls to GitHub).

It may be helpful to see what kind of Git information is already included in Hugo: https://gohugo.io/variables/git/#readout.

File: layouts/partials/main/footer.html (hunk 1 above)
Comment (PriyanshuRj) [reply layouts/partials/main/footer.html]: Okay sorry for the same

**Comment by PriyanshuRj** [comment]
New Look after adapting to Hugo .GitInfo
image

**Review by xavierchanth — CHANGES_REQUESTED** [review]
Looks good, but to avoid code duplication please put your work into the main/footer partial.
The file is at `layouts/partials/main/footer.html`

**Comment by PriyanshuRj** [comment]
> Looks good, but to avoid code duplication please put your work into the main/footer partial. The file is at `layouts/partials/main/footer.html`

Actually, the git variables are not accessible in the partials so we need to pass them down which will also create extra code.

**Comment by xavierchanth** [comment]
> > Looks good, but to avoid code duplication please put your work into the main/footer partial. The file is at `layouts/partials/main/footer.html`
> 
> Actually, the git variables are not accessible in the partials so we need to pass them down which will also create extra code.

Ah you are correct, the partial call doesn't have the context being passed down.
You can in pass down a `.` for everything, or just the Git variable context.

>One of the most common mistakes with new Hugo users is failing to pass a context to the partial call. In the pattern above, note how “the dot” (.) is required as the second argument to give the partial context. You can read more about “the dot” in the [Hugo templating introduction](https://gohugo.io/templates/introduction/).  
Source: https://gohugo.io/templates/partials/#use-partials-in-your-templates

This partial didn't have the context being passed, since it didn't need it prior to now.

**Comment by PriyanshuRj** [comment]
> is failing to pass a context to the partial call. In the pattern above,

ok got it thanks

**Comment by PriyanshuRj** [comment]
> > > Looks good, but to avoid code duplication please put your work into the main/footer partial. The file is at `layouts/partials/main/footer.html`
> > 
> > 
> > Actually, the git variables are not accessible in the partials so we need to pass them down which will also create extra code.
> 
> Ah you are correct, the partial call doesn't have the context being passed down. You can in pass down a `.` for everything, or just the Git variable context.
> 
> > One of the most common mistakes with new Hugo users is failing to pass a context to the partial call. In the pattern above, note how “the dot” (.) is required as the second argument to give the partial context. You can read more about “the dot” in the [Hugo templating introduction](https://gohugo.io/templates/introduction/).
> > Source: https://gohugo.io/templates/partials/#use-partials-in-your-templates
> 
> This partial didn't have the context being passed, since it didn't need it prior to now.

Thanks got it.

**Review by xavierchanth — APPROVED** [review]

**Review by TylerTrott — APPROVED** [review]

**Comment by JeremyTubongbanua** [comment]
Amazing work @PriyanshuRj :)

**Pull request merged**